stories/troubleshooting.stories.mdx

```diff
@@ -0,0 +1,59 @@
+<Meta title="Documentation/Troubleshooting"/>
+
+# Troubleshooting
+Looking to get involved in the mono repo but running into issues? Read on to find out how to fix them!
+
+## Getting started
+
+### First time setup
+Every time you get the latest changes in the mono repository, you need to follow a few short steps to get up and running
+
+1. Install yarn packages at the root: `yarn`
+2. Run the build command at the root: `yarn build`
+3. You can now run storybook using the command `yarn storybook:serve`
+
+### Cannot find module `@justeat/f-<any-package-name>`
+This issue indicates that you have not run yarn build with the latest code, refer to `First time setup` for a solution.
+
+### Storybook error reading: `ENOENT: no such file or directory, open ...`
+This issue indicates that you have not run yarn build with the latest code, refer to `First time setup` for a solution.
+
+## Development Process
+
+### I am unable to run the 'demo' script within a component
+- We recently changed the way interaction based tests are executed; they no longer depend on a demo script, but are now run within storybook
+- As a result of those changes, the demo script you may have previously used has been removed
+- If you find you have a use case where you need to demo a component outside of storybook; contact the team for assistance.
+
+### How can I test an unpublished version of my component in another project?
+Sometimes we might want to test changes to a component before we submit a PR. You can use yalc to do this.
+
+- Read more [about yalc and how to install it](https://github.com/whitecolor/yalc)
+- Yalc works by creating a locally published directory on your machine, and allows other code bases to use it in place of a real dependency
+- Once installed you can run `yalc publish --push` in any component
+- Once published locally you can run `yalc add @justeat/f-myComponentName` in any NPM based project
+- Assets should be automatically updated when you change the component
```

Comment:
This section isn't quite correct. You run `yalc publish` initially, then `yalc add @justeat/f-myComponentName` inside the consuming application to depend on it. If you then make further changes to the component then you run `yalc publish --push` inside the component and you don't have to re-add it.

```suggestion
- Once installed you can run `yalc publish` in any component
- Once published locally you can run `yalc add @justeat/f-myComponentName` in any NPM based project to add the local dependency
- If you update the component locally and need the dependency to be updated, run `yalc publish --push` inside the component. You may also need to restart your consuming application if the new changes aren't picked up by hot-module reloading (HMR).
```